parser_library/src/diagnostic.h

```diff
@@ -457,20 +458,20 @@ struct diagnostic_op
 
     static diagnostic_op error_M112(const std::string& instr_name, const range& range);
 
-    static diagnostic_op error_M113(const std::string& instr_name, const range& range);
```

Comment:
I think we should keep the original message ids, the messages relate to basically the same issues as before.